markets/perps-market/contracts/storage/PerpsMarketConfiguration.sol

```diff
@@ -99,8 +99,10 @@ library PerpsMarketConfiguration {
         maintenanceMarginRatio = impactOnSkew.mulDecimal(self.maintenanceMarginRatioD18);
 
         uint256 notional = sizeAbs.mulDecimal(price);
-        initialMargin = notional.mulDecimal(initialMarginRatio);
-        maintenanceMargin = notional.mulDecimal(maintenanceMarginRatio);
+        initialMargin = notional.mulDecimal(initialMarginRatio) + self.minimumPositionMargin;
```

Comment:
Why is the minimum added to every position instead of doing something like the max of the two values? Will this cause problems at really small margin values?

For example: If the minimum is $50 and the trader is opening a position which requires $50 in margin, won't this require them to have $100?